Containerfile

```diff
@@ -0,0 +1,37 @@
+ARG GO_VERSION=1.23
```

Comment:
Why don't use the current Go version 1.24?

Reply:
Although it's the minimum required Go version, our `go.mod` file specifies 1.23, and I wanted to maintain consistency with that. I don't have a strong preference regarding which version to use or if it's necessary at all.